### ISSUE TITLE
Link original papers and reproduction studies

### ISSUE THREAD

**Issue by evanmiltenburg** [acl-org/acl-anthology#3356]
Given the increasing attention for reproduction studies in NLP, it would be nice if the Anthology could indicate whether a study was reproduced. It's probably not a good idea to note whether the reproduction study was successful, because there are many different shades of reproductions. But creating connections between different papers seems useful.

**Comment by mbollmann** [comment]
Interesting idea! I agree that this is useful information, but my initial thought is that the Anthology may not be the right place to record this, as this goes far beyond metadata for a paper, and more towards a semantic linking of papers (“_paper-X_ reproduces _paper-Y_”). This seems more appropriate for a service like e.g. Semantic Scholar that already does AI-powered analysis of paper content and could therefore also extract this information in the first place.

Just my 2 cents.

**Comment by evanmiltenburg** [comment]
Right; the problem is that people go to the anthology for citation information. So if they don't see it there, they will not know about any replications. I guess this might be an argument for the community to adopt a service like pubpeer to add comments to papers.On 4 Jun 2024, at 17:51, Marcel Bollmann ***@***.***> wrote:﻿
Interesting idea! I agree that this is useful information, but my initial thought is that the Anthology may not be the right place to record this, as this goes far beyond metadata for a paper, and more towards a semantic linking of papers (“paper-X reproduces paper-Y”). This seems more appropriate for a service like e.g. Semantic Scholar that already does AI-powered analysis of paper content and could therefore also extract this information in the first place.
Just my 2 cents.

**Comment by mbollmann** [comment]
We can always integrate pointers to complementary resources, like we already do with Semantic Scholar or Papers with Code.

**Comment by nschneid** [comment]
I agree links between related papers can be useful but that is not something we have in the Anthology.

Closing this but feel free to submit a concrete suggestion like an integration with another service that helps navigate related publications.
